app/models/work.rb

```diff
@@ -0,0 +1,20 @@
+class Work < ApplicationRecord
+  has_many :votes
+
+  validates :title, presence: true
+
+  def self.get_spotlight
+    works = Work.all
+    return works.max_by { |work| work.votes.count }
+  end
+
+  def self.get_top_ten(category)
+    return self.sorted_works(category)[0...10]
+  end
```

Comment:
I love the way the methods in this file build on each other, each providing a small bit of functionality. This is an excellent example of functional decomposition.